safeeyes/core.py

```diff
@@ -111,59 +100,54 @@ def stop(self, is_resting=False):
             # Stop the break thread
             self.waiting_condition.acquire()
             self.running = False
-            if self.context['state'] != State.QUIT:
-                self.context['state'] = State.RESTING if (is_resting) else State.STOPPED
+            if self.context["state"] != State.QUIT:
+                self.context["state"] = State.RESTING if (is_resting) else State.STOPPED
             self.waiting_condition.notify_all()
             self.waiting_condition.release()
 
     def skip(self):
-        """
-        User skipped the break using Skip button
-        """
-        self.context['skipped'] = True
+        """User skipped the break using Skip button."""
+        self.context["skipped"] = True
 
     def postpone(self, duration=-1):
-        """
-        User postponed the break using Postpone button
-        """
+        """User postponed the break using Postpone button."""
         if duration > 0:
             self.postpone_duration = duration
         else:
             self.postpone_duration = self.default_postpone_duration
         logging.debug("Postpone the break for %d seconds", self.postpone_duration)
-        self.context['postponed'] = True
+        self.context["postponed"] = True
 
-    def get_break_time(self, break_type = None):
-        """
-        Returns the next break time
-        """
+    def get_break_time(self, break_type=None):
+        """Returns the next break time."""
         break_obj = self.break_queue.get_break(break_type)
         if not break_obj:
             return False
-        time = self.scheduled_next_break_time + datetime.timedelta(minutes=break_obj.time - self.break_queue.get_break().time)
+        time = self.scheduled_next_break_time + datetime.timedelta(
+            minutes=break_obj.time - self.break_queue.get_break().time
+        )
         return time
 
-    def take_break(self, break_type = None):
-        """
-        Calling this method stops the scheduler and show the next break screen
+    def take_break(self, break_type=None):
+        """Calling this method stops the scheduler and show the next break
```

Comment:
Not very readable, maybe limit docsring to some length and them divide it by lines? For example, there is first main line and after it - more detailed description - like `git commit message`

```python
"""Main short function description.

Some details.
"""

```

Reply:
This is what the formatting was supposed to do, in theory.
However, the line is also too long, so it was reformatted.

In general, I am not thrilled about the docstring changes in this PR.
I am considering reverting those, and also leaving them for a future PR.